data-raw/workflow_badge_urls_db.R

```diff
@@ -0,0 +1,51 @@
+extract_gha_workflows <- function(username,
+                                  repository,
+                                  folder_path = ".github/workflows",
+                                  token = Sys.getenv("GITHUB_PAT")) {
```

Comment:
What's the token we are using for this repo? I guess that's what we will need here?

Reply:
It is now one of Rémi's tokens, it is automatically retrieved in the action workflow.